content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -151,7 +145,7 @@ to access the mongoDB service.
 
     The step above does the following:
 
-    * Creates a `ServiceRole` "mongodb-viewer" which allows access to the port 27017 of `mongodb` service.
+    * Creates a `ServiceRole` "mongodb-viewer" which allows access to the port 27017 of MongoDB service.
```

Comment:
Do not use use the value of `kind:` in the CRD's to refer to to them. Use the simple terms instead: service role. I have commented on this before. This comment applies to all documentation. In the past, this has been done incorrectly by the wider community causing a lot of confusion. I am trying hard to drive the correct use. Missing article: ... of the MongoDB service.